expression/builtin_time_vec_test.go

```diff
@@ -98,6 +98,9 @@ var vecBuiltinTimeCases = map[string][]vecExprBenchCase{
 		},
 	},
 	ast.MakeTime: {},
+	ast.Extract: {
+		{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETString, types.ETDuration}},
```

Comment:
We can use selectStringGener to make arg[0] is one of time unit and make the generated data is legal.